static/app/views/detectors/detectorListRow.tsx

```diff
@@ -0,0 +1,150 @@
+import styled from '@emotion/styled';
+
+import Checkbox from 'sentry/components/checkbox';
+import {Flex} from 'sentry/components/container/flex';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import {
+  ConnectionCell,
+  type Item,
+} from 'sentry/components/workflowEngine/gridCell/connectionCell';
+import {EmptyCell} from 'sentry/components/workflowEngine/gridCell/emptyCell';
+import {IssueCell} from 'sentry/components/workflowEngine/gridCell/issueCell';
+import {NumberCell} from 'sentry/components/workflowEngine/gridCell/numberCell';
+import {TitleCell} from 'sentry/components/workflowEngine/gridCell/titleCell';
+import {tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import type {AvatarProject} from 'sentry/types/project';
+
+export type Detector = {
+  automations: Item[];
+  groups: Group[];
+  id: string;
+  link: string;
+  name: string;
+  project: AvatarProject;
```

Comment:
This could be `Partial<AvatarProject>` for now

Reply:
I used `AvatarProject` here since that's what `ProjectBadge` takes (`ProjectBadge` is used for the `TitleCell`)

https://github.com/getsentry/sentry/blob/5e42ea92ca93527b75c9e66ffad746a4a826d06b/static/app/components/idBadge/projectBadge.tsx#L15